relay-general/src/protocol/mod.rs

```diff
@@ -50,7 +50,9 @@ pub use self::request::{Cookies, HeaderName, HeaderValue, Headers, Query, Reques
 #[cfg(feature = "jsonschema")]
 pub use self::schema::event_json_schema;
 pub use self::security_report::{Csp, ExpectCt, ExpectStaple, Hpkp, SecurityReportType};
-pub use self::session::{ParseSessionStatusError, SessionAttributes, SessionStatus, SessionUpdate};
+pub use self::session::{
+    ParseSessionStatusError, SessionAggregates, SessionAttributes, SessionStatus, SessionUpdate,
```

Comment:
`SessionAggregateItem` should be exported here, since it is part of the public interface of `SessionAggregates`.